### PR TITLE
2nd PR done

### PR DESCRIPTION
**Pull request by secureamd**
my second pr

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,24 @@
...
+      [
+        trimRightStr(name, 10).padEnd(10),
+        ("+" + formatNum(additions)).padStart(7) +
+          "/" +
+          ("-" + formatNum(deletions)).padStart(7),
+        generateBarChart(percent, 21),
+      ].join(" ") +
+        percent.toFixed(1).padStart(5) +
+        "%"
+    );
```
Comment (sanishlodha) [inline README.md:20]: remove this line

Comment (sanishlodha) [inline README.md:20]: this one sorry

**Review by secureamd — COMMENTED** [review]
done

**Review by secureamd — COMMENTED** [review]
bola na done

File: README.md (hunk 1 above)
Comment (sanishlodha) [inline README.md:20]: please remove this line

**Review by sanishlodha — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (secureamd) [reply README.md:20]: ok done

File: README.md [hunk 2]
```diff
@@ -1 +1,24 @@
-# helloworld
\ No newline at end of file
+# helloworld
+
+```js
+const createContent = (languages) => {
+  const lines = [];
+  for (let i = 0; i < Math.min(languages.length, 5); i++) {
+    const data = languages[i];
+    const { name, percent, additions, deletions } = data;
```
Comment (secureamd) [inline README.md:8]: what about this?

**Review by sanishlodha — APPROVED** [review]

**Review by sanishlodha — APPROVED** [review]

**Pull request merged**